frontend/src/app/pages/exercises/exercise/shared/emergency-operations-center/transfer-target-dropdown/transfer-target-dropdown.component.html

```diff
@@ -0,0 +1,47 @@
+<ng-container *ngIf="transferPoints$ | async | values as transferPoints">
+    <button
+        ngbDropdownToggle
+        type="button"
+        class="btn btn-outline-primary btn-sm"
+        data-cy="sendAlarmGroupChooseTargetButton"
+        [disabled]="transferPoints.length === 0"
+    >
+        <app-transfer-point-name
+            *ngIf="selectedTransferPointId; else noTargetTransferPoint"
+            [transferPointId]="selectedTransferPointId"
+        >
+        </app-transfer-point-name>
```

Comment:
Use a self-closing tag.

Reply:
Done in 9fae3e2535a1c52601085fc6126acc7d4efa034b